scaleway/resource_object_bucket_test.go

```diff
@@ -391,32 +404,35 @@ func testAccCheckScalewayObjectBucketCors(tt *TestTools, n string, corsRules []*
 func testAccCheckScalewayObjectBucketExists(tt *TestTools, n string) resource.TestCheckFunc {
 	return func(state *terraform.State) error {
 		rs := state.RootModule().Resources[n]
-		bucketName := rs.Primary.Attributes["name"]
+		if rs != nil {
```

Comment:
Could you have the test of rs == nil return an error and then we avoid imbricating all that code in this conditional?